lnurl/explainer.md

```diff
@@ -0,0 +1,352 @@
+# LNURL
+
+LNURL is a standard for streamlining the process of enabling a client/consumer to conduct certain Lightning Network based operations with a web service that is connected to the Lightning Network. The client can be a wallet application, or even another web service. LNURL Flows automate some otherwise manual steps when executing these operations between two or more parties. This allows web service providers that make use of Lightning Network functionality to deploy API endpoints that can be interacted with autonomously by users implementing one or more of the various flows.
+
```

Comment:
We should highlight the benefits to the different parties.

```suggestion
### Service Provider
As a service provider, ... interoperability important and end-users using different clients...

### Client
As a client integrating LNURL ... streamline the user experience... since ... happens in the background ....

```

Reply:
> We should highlight the benefits to the different parties.

Not sure what these would be past the UX stuff already outlined in the doc. Interoperability with different clients already comes with LN being a protocol (for service providers), and the UX stuff is explained in the different sections (for clients).

What sorts of things you thinking?

Reply:
You know how when we came across the project it was difficult to grok how it can be used, who should be the different parties that can benefit from it. So the idea was just to give a one-liner for the benefits each of the actors can expect.

How do you convince a service provider, or a client to implement lnurl? If this is going to be blocking though I'd be happy to follow up in another PR. Eager to get more pages on the bolt.fun site. It brings me a special kind of joy when we publish there I'm not going to lie.